test/ActiveRecordSelectTest.php

```diff
@@ -67,4 +81,12 @@ public function testFindWithSelectNonSelectedFieldsShouldNotHaveAttributes()
         $author->id;
         $this->fail('expected ActiveRecord\UndefinedPropertyExecption');
     }
+
+    public function testReselect()
+    {
+        $this->expectException(UndefinedPropertyException::class);
+        $author = Author::select('name')->reselect('author_id')->first();
+        $author->name;
+        $this->fail('expected ActiveRecord\UndefinedPropertyExecption');
```

Comment:
I don't think you need this line; as a matter of fact, I think they can likely all be removed.

Reply:
I've copied and pasted `$this->fail('expected ActiveRecord\UndefinedPropertyExecption');` from other tests, but didn't know what they were there for. Removed.

Reply:
Yeah, that's my fault, I added all those `expectException` lines, but didn't catch all the `fail` calls. Thanks much.

Reply:
Thanks for clarifying.

I tracked the other extraneous $this->fail() functions and removed them in the latest commit

Reply:
Thanks much.